src/core/api/link_api.cpp

```diff
@@ -418,6 +424,17 @@ otError otLinkCslSetPeriod(otInstance *aInstance, uint16_t aPeriod)
     return error;
 }
 
+otError otLinkCslSetPeriodMs(otInstance *aInstance, uint32_t aPeriod)
+{
+    Error error = kErrorNone;
+
+    VerifyOrExit(aPeriod * 1000 / kUsPerTenSymbols <= NumericLimits<uint16_t>::kMax, error = kErrorInvalidArgs);
+    return otLinkCslSetPeriod(aInstance, static_cast<uint16_t>(aPeriod * 1000 / kUsPerTenSymbols));
```

Comment:
nit: single return statement
```suggestion
    error = otLinkCslSetPeriod(aInstance, static_cast<uint16_t>(aPeriod * 1000 / kUsPerTenSymbols));
```